README.md

```diff
@@ -30,21 +30,25 @@ QChat leverages event invokation to broadcast realtime messages across the chann
 
 ## Setup
 
-### McAfee OpenDXL SDK
+### Dependencies
 
-https://www.mcafee.com/us/developers/open-dxl/index.aspx
+Qchat only supports Linux Desktop systems.
```

Comment:
Ah, sorry, the dependency on curses is why I thought it was only supporting Linux. What library are you using in Windows to satisfy the curses dependency?